tests/coverage/reachable/assert-false/expected

```diff
@@ -0,0 +1,21 @@
+library/kani/src/arbitrary.rs, 55, COVERED
```

Comment:
Code coverage should exclude dependencies and only report coverage for user crates.

Reply:
I'm not sure I agree with this. If we exclude dependencies, won't it be more confusing for users who are debugging?

Reply:
AFAIK, code coverage tools exclude dependencies, because what users care about in testing is that their own code is covered. The target is to get to 100% code coverage with tests, which is likely going to be impossible if code in dependencies needs to be covered as well.

Reply:
I agree with @zhassan-aws's comments here, if we are calling it line coverage per harness, including coverage information about the library code makes it confusing.

Reply:
Whether coverage in dependencies counts towards certain coverage metrics or not is a different discussion. Note that users doing harness debugging may be interested in the coverage reported on external dependencies, and if we filter these results out they'll have no indication of where the trace went.

Reply:
Resolved offline. Left as an open question in the [RFC](https://github.com/model-checking/kani/blob/main/rfc/src/rfcs/0008-line-coverage.md#open-questions) but for now we are going to display dependency coverage information in the post-processed output as well.